src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
```

Comment:
The prefix 'ball' is unnecessary because these fields are already related to the entity Ball.
Pay attention to the requirements for the task.